docs/ABOUT.md

```diff
@@ -0,0 +1,14 @@
+Javascript is a scripting language used to provide dynamic and interactive content on webpages. It does not have class-oriented inheritance, but it does have prototype-oriented inheritance. JavaScript provides benefits like :- 
+* Less server interaction as in validating user input before sending page to server, thus, reducing server traffic.
+* Immediate feedback to the site visitors so that they dont reload page just to find they forgot to enter something.
+* Increased interactivity and Richer interfaces facilitating users to experience mouse hover changes, drag-drop components, animations etc.
+
+Client-Side JavaScript which enables the enhancement and manipulation of web pages and client browsers, is commonly used. The script can be referenced by or included in the HTML and code is interpreted by the browser. This takes the plain static HTML page to a level where user can interact to it.
```

Comment:
Maybe: "Client-side JavaScript is interpreted in the browser without requiring compilation.  This allows interactive content to be included in HTML pages."